modules/nf-core/pyrodigal/tests/main.nf.test

```diff
@@ -0,0 +1,30 @@
+nextflow_process {
+
+    name "Test Process PYRODIGAL"
+    script "../main.nf"
+    process "PYRODIGAL"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pyrodigal"
+
+    test("pyrodigal") {
+        when {
+            process {
+                """
+                input[0] =  [
+                             [id:'test', single_end:false ], // meta map
+                             file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                            ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+}
```

Comment:
Please also add a stub test, which we will probably require soon for nf-core modules.
You can see an example here: https://github.com/nf-core/modules/pull/4572/files

Reply:
@mashehu But i thought a stub test is required only if a module relies on large databases for the test data like mentioned in the [DSL2 documentation](https://nf-co.re/docs/contributing/modules#migrating-from-pytest-to-nf-test). Pyrodigal doesnt need such large test data.

Reply:
One use case for stubs is as you describe, if we can't run the normal tests. Another one is to quickly go through a whole pipline as outlined here: https://github.com/nf-core/modules/issues/4570
It's fine as it is, but might change soon™️

Reply:
Hey @mashehu . Thanks for the explanation. 
In that case i just added the stub section to pyrodigal. Would you please have a quick look again? Much appreciated.